packages/steps/StepRegistry.js

```diff
@@ -0,0 +1,11 @@
+function StepRegistry() {
```

Comment:
IMHO this small amount of code is not worth an separate module. WDYT to put that into workorder module